packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -58,6 +58,8 @@ oracleAdaptersProxyPackage = "reya-oracle-adapters:1.0.1@proxy"
 oracleAdaptersRouterPackage = "reya-oracle-adapters:1.0.7@router"
```

Comment:
include sbt mainnet.toml above

Reply:
added in [3132942](https://github.com/Reya-Labs/reya-deployments/pull/259/commits/31329427253584851c29f016797476ccfc7cc92d)